app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Turniquette</title>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
```

Comment:
I was thinking on sending an authtoken in each request. So that I don't see why it should be usefull, we'll discuss it...